packages/web-client/app/utils/token.ts

```diff
@@ -1,6 +1,15 @@
-import { BigNumber } from '@ethersproject/bignumber/lib/bignumber';
+// ideally we would want to just use parseEther from @ethersproject/units
+// but there seems to be a typescript incompatibility between BigNumber produced by that
+// and BigNumber imported from @ethersproject/bignumber
+import { BigNumber, FixedNumber } from '@ethersproject/bignumber';
 
-export function amountInWei(amount: number, decimals: number = 18) {
-  let dec = BigNumber.from(10).pow(BigNumber.from(decimals));
-  return BigNumber.from(amount).mul(dec);
+export function amountInWei(amount: FixedNumber, decimals: number = 18) {
```

Comment:
`amountInWei` seems to only be used in a few places that receive the value to relay. Though we're storing this value as a string, I didn't feel good passing a string to this function. Passing a `FixedNumber` felt like a guarantee of more structure by the time it got here - though I'm not very sure if this is true.